src/main/resources/com/google/api/codegen/csharp/gapic_client.snip

```diff
@@ -526,6 +546,52 @@
 @end
 
 # api: StaticLangApiView, method: StaticLangApiMethodView, isImp: boolean
+@private streamingRequestObjectMethodServer(api, method, isImpl)
+    @if isImpl
+        public override {@method.name}Stream {@method.name}(
```

Comment:
The underlying call here is actually asynchronous. I agree this is slightly unexpected.
The return type of the gRPC call is AsyncServerStreamingCall, and it performs an async request.